docs/dev/guidelines/server.rst

```diff
@@ -483,6 +483,15 @@ The table contains all annotations for the corresponding minimum role including
 
 If, for some reason, you need to deviate from these rules, use ``@ManualConfig``. Use this annotation only if absolutely necessary as it will exclude the endpoint from the automatic authorization tests.
 
+Use the ``@Internal`` annotation to mark methods or classes that should only be accessed from trusted internal networks.
+Access to these endpoints is restricted based on the configured CIDR blocks in ``InternalAccessConfiguration`` (``artemis.security.internal.allowed-cidrs``).
```

Comment:
can you write CIDR once completely and link to an external resource what it means? It would also be good to add a short example here or reference one in the code